myapp/app/controllers/tasks_controller.rb

```diff
@@ -3,7 +3,7 @@ class TasksController < ApplicationController
   before_action :get_users, only: [:new, :edit]
 
   def index
-    @tasks = current_user.tasks.eager_load(:user).all.page(params[:page])
+    @tasks = current_user.tasks.includes([:labellings, :labels]).all.page(params[:page])
```

Comment:
`tasks` と `labellings` と `labels` の関係性を教えて欲しいです。
`labellings`が中間テーブルの役割をしていたと思っていたのですが、違いましたかね？
その場合だと`has_many`と`belongs_to`って下記のページのようになるはずかなと考えてます。
https://railsguides.jp/association_basics.html#belongs-to%E3%81%AE%E3%82%B9%E3%82%B3%E3%83%BC%E3%83%97-includes

そして、`includes`も中間テーブルをかいして取得してくる場合の記述のしかたが下記になるので少し気になりました。
```suggestion
    @tasks = current_user.tasks.includes([labellings: :labels]).all.page(params[:page])
```

設計書のモデル図も記載なかったので一旦設計を確認させて欲しいです！

Reply:
失礼しました、READMEにモデル図を記載しました。
（ソースはまだ修正していないです。）

テーブルの関係性としては、labellingsが中間テーブルでtasksとlabelsが多対多でlabellingsでtaskとlabelの紐づき情報を持っています。
そのため、タスクに複数のラベルが設定されている場合は、その設定分、labellingsにレコードが作成される仕様にしています。

includesの記述の仕方については確認いたします。

Reply:
あー失礼しました！
`tasks`と`labels`は多対多の関係ということですね！
<img width="641" alt="スクリーンショット 2022-09-15 11 44 14" src="https://user-images.githubusercontent.com/46880814/190301387-836f9cc5-7550-4957-8b6c-823d7e496920.png">

上記の図で認識あっていれば大丈夫そうです！
池田の指摘は無視してください！

Reply:
上記の認識です！
すみません、READMEのモデル図更新が漏れていたためお手間をとらせてしまいました。